erpnext/projects/doctype/task/task.py

```diff
@@ -201,6 +201,9 @@ def set_multiple_status(names, status):
 def set_tasks_as_overdue():
 	tasks = frappe.get_all("Task", filters={'status':['not in',['Cancelled', 'Closed']]})
 	for task in tasks:
+		if frappe.db.get_value("Task", task.name, "status") in 'Pending Approval':
```

Comment:
'Pending Approval' is not a valid status of task

Reply:
I will change